modulefiles/module_base.orion.lua

```diff
@@ -0,0 +1,35 @@
+help([[
+Load environment to run GFS on Orion
+]])
+
+prepend_path("MODULEPATH", "/apps/contrib/NCEP/libs/hpc-stack/modulefiles/stack")
+
+load(pathJoin("hpc", os.getenv("hpc_ver")))
+load(pathJoin("hpc-intel", os.getenv("hpc_intel_ver")))
+load(pathJoin("hpc-impi", os.getenv("hpc_impi_ver")))
+
+load(pathJoin("esmf", os.getenv("esmf_ver")))
+
+load(pathJoin("python", os.getenv("python_ver")))
+load(pathJoin("gempak", os.getenv("gempak_ver")))
+load(pathJoin("perl", os.getenv("perl_ver")))
```

Comment:
perl, libjpeg, udunits, bufr_dump, and util_shared weren't loaded in TCL version

Reply:
See reply to above comment about `module_base.hera.lua`.